test/LidoStakedEthRewardsIntegration.t.sol

```diff
@@ -0,0 +1,194 @@
+        uint256 whaleReward1 = 5.028734135612479150 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), whaleReward1);
+        assertEq(IERC20(WSTETH).balanceOf(operator),     REWARD_AMOUNT - whaleReward1);
+
+        skip(DURATION - skipAmount);
+
+        // 7-siblings wallet should get about 50% of the total rewards ~10 wstETH
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress);
+        uint256 whaleReward2 = 10.057468271224958300 ether;
+        assertEq(whaleReward2, whaleReward1 * 2);
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), whaleReward2);
+        assertEq(IERC20(WSTETH).balanceOf(operator),     REWARD_AMOUNT - whaleReward2);
+
+        skip(DURATION);  // Skip twice the rewards period
+
+        // 7-siblings should receive no more rewards
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress);
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), whaleReward2);
+        assertEq(IERC20(WSTETH).balanceOf(operator),     REWARD_AMOUNT - whaleReward2);
+    }
+
+    function test_multiple_users_claim() public {
+        address claimAddress1 = makeAddr("claimAddress1");
+        address claimAddress2 = makeAddr("claimAddress2");
+        address[] memory assets = new address[](1);
+        assets[0] = _getAToken(WETH);
+
+        _setupDistribution();
+
+        uint256 skipAmount = DURATION / 3;  // 33% of rewards distributed
+        skip(skipAmount);
+
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress1);
+        uint256 whale1Reward1 = 3.352489423741600545 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress1), whale1Reward1);
+        assertEq(IERC20(WSTETH).balanceOf(operator),      REWARD_AMOUNT - whale1Reward1);
+
+        vm.prank(whale2);
+        incentives.claimAllRewards(assets, claimAddress2);
+        uint256 whale2Reward1 = 0.423580642569205639 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress2), whale2Reward1);
+        assertEq(IERC20(WSTETH).balanceOf(operator),      REWARD_AMOUNT - whale1Reward1 - whale2Reward1);
+
+        skip(DURATION);  // Skip past the end of the rewards period
+
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress1);
+        uint256 whale1Reward2 = 10.057468271224879968 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress1), whale1Reward2);
+        assertEq(IERC20(WSTETH).balanceOf(operator),      REWARD_AMOUNT - whale1Reward2 - whale2Reward1);
+
+        vm.prank(whale2);
+        incentives.claimAllRewards(assets, claimAddress2);
+        uint256 whale2Reward2 = 1.270741927707626814 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress2), whale2Reward2);
+        assertEq(IERC20(WSTETH).balanceOf(operator),      REWARD_AMOUNT - whale1Reward2 - whale2Reward2);
+        assertEq(IERC20(WSTETH).balanceOf(operator),      8.671789801067493218 ether);
```

Comment:
I'd move `8.671789801067493218 ether` to a variable with a meaningful name - similarly to every other ether constant in the code.

Reply:
Fixed. Are any other names unclear?